extensions/wikia/Rail/templates/RailController_Index.php

```diff
@@ -1,25 +1,27 @@
 <? if ( $isAside ): ?>
 	<aside>
 <? endif; ?>
-	<div id="WikiaRail" class="WikiaRail<?= !empty($isGridLayoutEnabled) ? ' grid-2' : '' ?>">
+	<div id="WikiaRailWrapper" class="WikiaRail<?= !empty($isGridLayoutEnabled) ? ' grid-2' : '' ?>">
```

Comment:
and it can't be done without additional wrapper?

From what I see in code the you use the WikiaRailWrapper to take it's height and offset top. It didn't work without additional wrapper?